decidim-blogs/app/commands/decidim/blogs/admin/create_post.rb

```diff
@@ -32,7 +32,7 @@ def create_post!
             title: @form.title,
             body: @form.body,
             component: @form.current_component,
-            author: @current_user
+            author: @form.user_group || @current_user
```

Comment:
Adding an `author` method to the form with this logic seems a _cleaner_ approach

Reply:
Added method

Reply:
Oh, I meant if we could move it to the form, rather than the command 😅

Reply:
Whoops. Yes that makes more sense :bowtie: